apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -1,17 +1,65 @@
 import test from "../../fixtures/common";
 import { Account } from "../../enum/Account";
-import { Transaction } from "../../models/Transaction";
+import { Token } from "../../enum/Token";
+import { Transaction, TokenTransaction } from "../../models/Transaction";
 import { specs } from "../../utils/speculos";
 import { Application } from "tests/page";
+import { addTmsLink } from "tests/fixtures/common";
 
-// ONLY TESTNET (SEND WILL BE APPROVED ON DEVICE)
 const transactions = [
   //TODO: Reactivate when fees will be stable
   //new Transaction(Account.tBTC_1, Account.tBTC_2, "0.00001", "medium"),
-  new Transaction(Account.sep_ETH_1, Account.sep_ETH_2.address, "0.00001", "medium"),
+  new Transaction(Account.sep_ETH_1, Account.sep_ETH_2, "0.00001", "medium"),
 ];
 
-//This test might sporadically fail due to getAppAndVersion issue - Jira: LIVE-12581
+const transactionETH_BTC = [new Transaction(Account.ETH_1, Account.BTC_1, "0.00001", "medium")];
+
+const transactionsToken = [
+  new TokenTransaction(
+    Account.ALGO_1,
+    Account.ALGO_2,
+    Account.ALGO_3,
+    "0.1",
+    "medium",
+    Token.ALGO_USDT,
+  ),
+];
+
+const transactionInputValid = [
+  new Transaction(Account.ETH_1, Account.ETH_2, "send max", "medium"),
+  new Transaction(Account.ETH_1, Account.ETH_2, "0.00001", "medium"),
+];
+
+const transactionsInputsInvalid = [
+  //Todo: add Dot Delegating account - warningMessage: Polkadot all funds warning
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "", "medium"),
+    expectedErrorMessage: null,
+  },
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "0", "medium"),
+    expectedErrorMessage: null,
+  },
+  {
+    transaction: new Transaction(Account.XRP_1, Account.XRP_2, "1", "medium"),
+    expectedErrorMessage: "Recipient address is inactive. Send at least 10 XRP to activate it",
+  },
+  {
+    transaction: new Transaction(Account.DOT_1, Account.DOT_2, "1.2", "medium"),
+    expectedErrorMessage: "Balance cannot be below 1 DOT. Send max to empty account.",
+  },
+  {
+    transaction: new Transaction(Account.DOT_1, Account.DOT_3, "0.5", "medium"),
+    expectedErrorMessage: "Recipient address is inactive. Send at least 1 DOT to activate it",
+  },
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "100", "medium"),
+    expectedErrorMessage: "Sorry, insufficient funds",
+  },
+];
+
+//This tests might sporadically fail due to getAppAndVersion issue - Jira: LIVE-12581
+
 for (const [i, transaction] of transactions.entries()) {
```

Comment:
I don't think we need data driven test if there is only one value

Reply:
For now there is only 1 Value but in the futur I planned to add more (BTC...)